Tests added

Cammelen opened the pull request:
Выполнены следующие тесты:

1. Тесты на регистрацию
2. Тесты на вход
3. Тесты на переходы в личном кабинете и выход из аккаунта
4. Тесты на переходы в конструкторе

Commit:
Tests added

nikolay-977 commented on pom.xml:
```diff
@@ -11,6 +11,90 @@
     <properties>
```
Можно улучшить: хорошей практикой является добавление README.md с описанием проекта, которое включает название, технологии в проекте (например, Java 11, JUnit 4.13.2, maven 3.9.0 и т.д.), настройка (если необходима) и как запускать (mvn clean test). Как оформлять README.md можно ознакомиться в статье https://dzen.ru/a/YbzudflBPE1IDJF-

Cammelen replied:
Файл README.md был добавлен в "Tests added"
https://github.com/Cammelen/Diplom_3/pull/1/commits/a8406e54bb80075aaf2a99975dffa398a79aa4ee

nikolay-977 commented on src/main/java/pageobject/Constructor.java:
```diff
@@ -0,0 +1,22 @@
+        this.webDriver = webDriver;
+    }
+
+    public By enterConstructor = By.xpath("//*[contains(text(), 'Конструктор')]");
```
Необходимо исправить: В начале класса следует описывать переменные, а затем конструкторы и методы. Также можно установить курсор на корневую директорию и нажать сочетание клавиш Ctrl+Alt+L (Option+Command+L для Mac), затем выбрать checkbox rearrange entries и нажать ok.

Cammelen replied:
Исправлено в "Second_commit"
https://github.com/Cammelen/Diplom_3/pull/1/commits/c73f2d72654fd88702eb90dcaf1e7426e5d66a53

nikolay-977 commented on src/main/java/pageobject/Constructor.java:
```diff
@@ -0,0 +1,22 @@
+
+    public By enterConstructor = By.xpath("//*[contains(text(), 'Конструктор')]");
+    public By enterLogo = By.xpath("//*[@xmlns = 'http://www.w3.org/2000/svg']");
+    public By toFillings = By.xpath("//*[contains(text(), 'Начинки')]");
```
Необходимо исправить: Для веб-элементов следует указывать модификатор доступа private, а для взаимодействия с ними использовать методы(шаги).

Cammelen replied:
Исправлено в "Second_commit"
https://github.com/Cammelen/Diplom_3/pull/1/commits/c73f2d72654fd88702eb90dcaf1e7426e5d66a53

Commit:
Second_commit

1. Исправлена структура в классах по порядку - переменные/конструкторы/методы
2. Для всех локаторов установлен доступ private и прописаны методы для их использования
